pkg/flag/registry_flags.go

```diff
@@ -65,6 +75,14 @@ func (f *RegistryFlagGroup) ToOptions() (RegistryOptions, error) {
 	var credentials []types.Credential
 	users := f.Username.Value()
 	passwords := f.Password.Value()
+	if f.PasswordStdin.Value() {
+		contents, err := io.ReadAll(os.Stdin)
```

Comment:
```suggestion
		if len(passwords) != 0 {
			log.Warnf("Using `--password` and `--password-stdin` at the same time is not supported. The password from `--password-stdin` is used.")
		}
		contents, err := io.ReadAll(os.Stdin)
```

Reply:
Nice catch. It's better to exit in that case. Fixed in https://github.com/aquasecurity/trivy/pull/7664/commits/5a4f9cb88e711a209fea51a5e889d64b92ca28ab